README.md

```diff
@@ -86,6 +86,17 @@ for each table that includes a `resource_owner_id` column:
 add_foreign_key :table_name, :users, column: :resource_owner_id
 ```
 
+Remember to add associations to your model so the related records are deleted.
+If you don't do this a `ActiveRecord::InvalidForeignKey`-error will be raised
+when you try to delete a model with related access grants or access tokens.
+
+```ruby
+class User < ApplicationRecord
+  has_many :oauth_access_grants, class_name: "Doorkeeper::AccessGrant", foreign_key: :resource_owner_id, dependent: :destroy
```

Comment:
Does it really required to name associations like `oauth_access_grants` ? `Doorkeeper::AccessGrant` already has table name prefixes, so Rails must recognize the name of the table for associations.

Maybe it will be better to name them like:

```ruby
has_many :access_grants, ...
has_many :access_tokens, ...
```
Also I suggest to change `dependent: :destroy` option to `dependent: :delete_all` - it will delete all the associated records in one SQL query instead of N+1. But we can save :destroy variant in comment, like " _# or :destroy if you need callbacks_"

What do you think?